broker/benches/neon_broker_bench.rs

```diff
@@ -0,0 +1,174 @@
+use std::sync::atomic::{AtomicU64, Ordering};
```

Comment:
NIT: the file can be named something like `timeline_pubsub` or something similar. `_bench` suffix is redundant to me since now fole lives under `/benches` and `neon_broker` too because everything happens under /broker

Reply:
Renamed to rps.rs, because that's what it really tests.